polygeist/tools/cgeist/Test/polybench/datamining/correlation/correlation.c

```diff
@@ -5,7 +5,7 @@
 // RUN: rm -f %s.out1 %s.out2
 // RUN: cgeist %s %polyexec %stdinclude -O3 -o %s.execm && %s.execm > %s.mlir.time; cat %s.mlir.time | FileCheck %s --check-prefix EXEC
 // RUN: clang %s -O3 %polyexec %stdinclude -o %s.exec2 -lm && %s.exec2 > %s.clang.time; cat %s.clang.time | FileCheck %s --check-prefix EXEC
-// RUN: rm -f %s.exec2 %s.execm
+// RUN: rm -f %s.exec2 %s.execm %s.mlir.time %s.clang.time
```

Comment:
Do you know why these tests us %s.* instead of %t.* for generated files?

Reply:
No I do not know, these are not our tests they come from the polygeist project.